src/js/_enqueues/wp/wp-image-mime-fallback-loader.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @output wp-includes/js/wp-image-mime-fallback-loader.js
+ */
+
+( function( document, image_src, settings ) {
+
+	var image = document.createElement( 'image' );
+	image.src = image_src + 'jIAAABXRUJQVlA4ICYAAACyAgCdASoCAAEALmk0mk0iIiIiIgBoSygABc6zbAAA/v56QAAAAA==';
+	image.onload = function() {
+		image.onload = undefined;
+		image.src = image_src + 'h4AAABXRUJQVlA4TBEAAAAvAQAAAAfQ//73v/+BiOh/AAA=';
```

Comment:
We should probably add comments here what these two values do and mean. @eugene-manuilov Since I believe you implemented this in the plugin, could you clarify? If you took this from another web resource, could you share the URL? It may be helpful to refer to that URL here too.

Reply:
The approach I looked at for detection is described here: https://developers.google.com/speed/webp/faq#in_your_own_javascript and these are tiny base64 encoded WebP images... however these strings seem to come from  https://gist.github.com/jcayzac/8345885 ; we should only need to detect support for one type of WebP (lossy) for this, this looks like it detects support for two formats, I'm not sure which. `"data:image/webp;base64,UklGRhoAAABXRUJQVlA4TA0AAAAvAAAAEAcQERGIiP4HAA=="` is the string used for lossy support detection in the original code

Reply:
I have added the comment on this image sources. As per this https://developers.google.com/speed/webp/gallery2 if the image has bigger size then it's Lossless otherwise it's lossy.

Attach screenshot for more information.

Lossy: https://paste.pics/f9f9a9b9ca44b9518409996b3725393d
Lossless: https://paste.pics/6b0d2c2e679a2cc46743f08ddb00b75c